src/about.html

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I feel this is, currently, your most successful page (on desktop) and definitely makes me wish the project had gotten closer to completion. I would also encourage you to explore the text treatment on this page a bit more in depth; how can the readability of the white text be at its best?